app/services/submission_service.rb

```diff
@@ -39,7 +39,7 @@ def create_submission(
         )
       end
 
-      submission = Submission.create!(create_params)
+      submission = Submission.create!(create_params).reload
```

Comment:
**non-blocking**: let's add a comment here 😄  to show other people why we added `reload`

Reply:
@kiryl-zubarau please check my last PR. I've created a module which is responsible for automatic reload of uuid field